Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PIDGain.mo

```diff
@@ -0,0 +1,95 @@
+﻿within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO;
+block PIDGain "Identify the control gain of a PID controller"
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput kp(final min=1E-6)
+    "Gain of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput T(final min=0)
+    "Time constant of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput L(final min=1E-6)
+    "Time delay of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
+        iconTransformation(extent={{-140,-80},{-100,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput k
+    "Control gain signal"
+    annotation (Placement(transformation(extent={{100,-20},{140,20}})));
+  Buildings.Controls.OBC.CDL.Continuous.Divide div1
+    "Calculate the inverse of the input gain"
+    annotation (Placement(transformation(extent={{-38,40},{-18,80}})));
```

Comment:
`extent={{-38,40},{-18,80}}` to `extent={{-38,40},{-18,60}}`

Reply:
changed it